src/posthog-core.js

```diff
@@ -1093,9 +1094,43 @@ PostHogLib.prototype.set_config = function (config) {
             this['config']['debug'] = true
         }
         Config.DEBUG = Config.DEBUG || this.get_config('debug')
+
+        if (this.sessionRecording && typeof config.disable_session_recording !== 'undefined') {
```

Comment:
Why are we verifying `this.sessionRecording`? Isn't it always going to be there? If not then this functionality does not work since sometimes starting does not start anything.

Reply:
I believe the first time this is called (by us on `init`) `this.sessionRecording` will be undefined. We manually start session recording ourselves on that case. This should mean that it will work as expected after `init`, which is the case with all other methods